ACE/NEWS

```diff
@@ -1,6 +1,10 @@
 USER VISIBLE CHANGES BETWEEN ACE-7.1.4 and ACE-7.1.5
```

Comment:
Please make this a major release if you're requiring a newer C++ standard.

Reply:
I will make it a new micro, we are not deprecating specific compilers, just lifting the required C++ standard

Reply:
Stop resolving this. I still don't agree.